sdk/cognitivelanguage/tests.yml

```diff
@@ -4,6 +4,9 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     EnvVars:
+      CONVERSATIONS_KEY: $(clu-key)
+      CONVERSATIONS_PROJECT: antischOne
```

Comment:
Is antischOne temporary?

Reply:
Temporary for a while, yes. We've been working with several teams across the Language pillar to come up with a unified plan and are pretty close. We'll eventually get these resources provisions with test-resources.json + a test-resources-post.ps1 to create, populate, and train projects.

@AhmedLeithy, one other idea is rather than the postscript, we could also do what Search does like so if that's easier: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/tests/Utilities/SearchResources.cs